<?xml version="1.0"?>

<launch>
    <arg name="waypoints_file" default="$(find waypoint_navigation)/waypoints_cfg/waypoints.yaml"/>
    <arg name="save_joy_button" default="0"/>
    <arg name="default_rad" default="0.8"/>


    <node name="waypoint_saver" pkg="waypoint_navigation" type="waypoint_saver" output="screen">
        <param name="save_joy_button"       value="$(arg save_joy_button)"/>
        <param name="filename"              value="$(arg waypoints_file)"/>
        <param name="default_rad"           value="$(arg default_rad)"/>
        
        <remap from="waypoints_joy"     to="joy"/>
        <remap from="waypoints_viz"     to="clicked_point"/>
        <remap from="finish_pose"       to="move_base_simple/goal"/>
    </node>

</launch>